addons/multiplayer/NetworkGame.gd

```diff
@@ -30,24 +30,31 @@ func connect_via_cli():
 				'--ip': ip = key_value[1]
 				'--port': port = int(key_value[1])
 	
-	if "--client" in OS.get_cmdline_args() or OS.get_environment("USE_CLIENT") == "true":
+	if is_on_web() or "--client" in OS.get_cmdline_args() or OS.get_environment("USE_CLIENT") == "true":
 		connect_client(ip, port)
 	else:
 		connect_server(port, "--dedicated" in OS.get_cmdline_args())
 
 func connect_client(ip, port):
-	var peer = NetworkedMultiplayerENet.new()
-	assert(peer.create_client(ip, port) == OK)
+	var peer = create_client(ip, port)
 	assert(get_tree().connect("server_disconnected", self, "client_server_gone") == OK)
 	get_tree().set_network_peer(peer)
 	
 	if change_window_title:
 		append_title_string(" (Client)")
 
+func create_client(ip, port):
+	if is_on_web():
+		var peer = WebSocketClient.new()
+		assert(peer.connect_to_url(ip + ":" + str(port), PoolStringArray(), true) == OK)
```

Comment:
I assume the protocol is when added automatically as ws:// when protocols is left empty?

Reply:
Yes, I just looked [into](https://github.com/godotengine/godot/blob/42d3993446a6a1367469cf6ffb91fad3ea79bf4f/modules/websocket/websocket_client.cpp#L41) [the](https://github.com/godotengine/godot/blob/42d3993446a6a1367469cf6ffb91fad3ea79bf4f/modules/websocket/emws_client.cpp#L66) [source](https://github.com/godotengine/godot/blob/42d3993446a6a1367469cf6ffb91fad3ea79bf4f/modules/websocket/library_godot_websocket.js#L148) and the protocol is alwas `ws://` or, if you specify it in the `url`, `wss://`. `protocols` only requests subprotocols (as described [here](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/WebSocket)).